mriqc/interfaces/bids.py

```diff
@@ -44,6 +44,11 @@ class ReadSidecarJSON(SimpleInterface):
     def _run_interface(self, runtime):
         metadata = get_metadata_for_nifti(self.inputs.in_file)
         output_keys = [key for key in list(self.output_spec().get().keys()) if key.endswith('_id')]
+
+        #in case no json sidecar files exist (ie no BIDS tree)
+        if  self.expr.search(op.basename(self.inputs.in_file))==None:
```

Comment:
if it is called with a single input file, and there is no sidecar Json, then I get a bug in the line after since the inputs.in_file is None  (it gives an error when .groudict() is called)
...
those 2 lines allow this function to be called even if there is no input ...
...
If you still do not want to modify it, then I guess I will have to prevent the ReadSidecaJSON to be add in the workflow. (line 234 in anatomical.py) this will make 2 workflows (but I do not know how to test if the corresponding json file exist) ... I find it more complex than those 2 lines ...